src/main/update.ts

```
@@ -0,0 +1,42 @@
+    detail: "A new version has been downloaded. Restart the application to apply the updates.",
+    restartButtonText: "Restart",
+    laterButtonText: "Later",
+  };
+
+  const assignedDialog = Object.assign({}, defaultDialogMessages, dialogProps);
+
+  return (info: IUpdateInfo, callback?: (response) => void) => {
+    const { releaseNotes, releaseName } = info;
+    const { title, restartButtonText, laterButtonText, detail } = assignedDialog;
+
+    const dialogOpts: Electron.MessageBoxOptions = {
```

ESLint

[error] 'Electron' is not defined.

(no-undef)